client/cmd/dexc/main.go

```diff
@@ -59,25 +59,29 @@ func main() {
 	clientCore, err := core.New(&core.Config{
 		DBPath:      cfg.DBPath, // global set in config.go
 		LoggerMaker: logMaker,
-		Certs:       cfg.Certs,
 		Net:         cfg.Net,
 	})
 	if err != nil {
-		fmt.Fprint(os.Stderr, "error creating client core: ", err)
+		fmt.Fprintf(os.Stderr, "error creating client core: %v\n", err)
 		os.Exit(1)
 	}
+
 	var wg sync.WaitGroup
 	wg.Add(1)
 	go func() {
 		clientCore.Run(appCtx)
 		wg.Done()
 	}()
+
 	// If explicitly running without web server then you must run the rpc
 	// server or the terminal ui.
 	if cfg.NoWeb && !cfg.RPCOn {
 		fmt.Fprintf(os.Stderr, "Cannot run without web server unless --rpc or --tui is specified\n")
 		os.Exit(1)
 	}
```

Comment:
It would make sense to move this config check and exit further up too, before `Core` is created or `TUI` checked.